src/Mod/Arch/ArchSectionPlane.py

```diff
@@ -705,28 +722,28 @@ def getCoinSVG(cutplane,objs,cameradata=None,linewidth=0.2,singleface=False,face
     lset = coin.SoIndexedLineSet()
     lset.coordIndex.setValues(0,2,[0,1])
     sep.addChild(lset)
-    rn.insertChild(sep,0)
+    rootNode.insertChild(sep,0)
 
     # set scenegraph
-    vv.setSceneGraph(rn)
+    inventorView.setSceneGraph(rootNode)
```

Comment:
We are trying to use a more Pythonic style according to our coding standards, which means using `snake_case` variables. I know most of the code is not like this at the moment, but we do try to change things little by little.

Reply:
@vocx-fc done